index.js

```diff
@@ -158,34 +166,49 @@ function addContext(url, context) {
 }
 
 async function createVcExamples() {
-  // generate base keypair and signature suite
-  const keyPairEd25519VerificationKey2020 = await Ed25519VerificationKey2020.generate();
-  const keyPairEd25519Multikey = await Ed25519Multikey.from(keyPairEd25519VerificationKey2020);
+  // generate base keypair and signature suites
+  // ecdsa-sd-2023
+  const keyPairEcdsaMultikeyKeyPair = await EcdsaMultikey
+    .generate({curve: 'P-256'});
+  const keyPairEcdsaMultiKey = await EcdsaMultikey
+    .from({...keyPairEcdsaMultikeyKeyPair});
```

Comment:
I don't think this step is necessary, you've already got the key pair above you can just use that to get `.signer()` from it.

Reply:
Fixed in a22b911